### ISSUE TITLE
Relationship between Protocols and object types

### ISSUE THREAD

**Issue by tlodderstedt** [openwallet-foundation/architecture-task-force#22]
I feel the verbs in the "Protocols" box are tied to certain types of objects (e.g. Withdrawal is related to Financial Instruments, I guess). Can we somehow preserve this relationship?

**Comment by OR13** [comment]
Yes, we talked about a model where the protocols might go under each object as a "method", in UML.

More discussion is needed.
